soda/scientific/soda/scientific/distribution/generate_dro.py

```diff
@@ -33,11 +62,45 @@ def generate_continuous_dro(self) -> DRO:
             data = data[~np.isnan(data)]
             logging.warning(
                 f"""{none_count} out of {data_len} rows ({none_count/data_len}%)
```

Comment:
Will this ever fail with divide by zero? or `data_len` is guaranteed to be non-zero?